app/controllers/users_controller.rb

```diff
@@ -1,5 +1,10 @@
 class UsersController < ApplicationController
+
   def show
+    render locals: {
+      facade: UserShowFacade.new(current_user),
+      token: current_user.github_token
```

Comment:
This works, I wonder if we can set an attr_reader for @token in UserShowFacade, and just call `if facade.token` in the view.